contract/test/TestAuditLog.sol

```diff
@@ -2,32 +2,103 @@ pragma solidity 0.5.x;
 
 import "truffle/Assert.sol";
 import "truffle/DeployedAddresses.sol";
-import "../contracts/auditLog.sol";
+import "../contracts/authRole.sol";
 
 contract TestAuditLog {
+    string skOwnerTest = "e0302f799ee2e6bcff366cf395dda8225e0a3ae9250740aeabf8e174a8d55c03";
+    string skMaintainerTest = "aaf4f5s4df5sd47h2ns4i54yk24a65sg7jt7iagj8lo4lj55wfwefjjjukaewqq4";
     string tableNameTest = "TableName";
     address addrTest = 0xc0ffee254729296a45a3885639AC7E10F9d54979;
+    enum ROLE {NULL, VIEWER, MAINTAINER, OWNER}
```

Comment:
The role might be a bit misleading. How can we distinguish from maintainer and owner?

Reply:
MAINTAINER can update the table only. However, a OWNER has the highest permission to do anythhing.

Reply:
I have mentioned the permission in the Postman document

Reply:
At least, append comments which describe the roles.